packages/expo-image/src/Image.tsx

```diff
@@ -69,14 +79,35 @@ export default class Image extends React.Component<ImageProps, ImageState> {
   }
 
   /**
-   * **Available on @Android only.** Caching the image that can be later used in ImageView
+   * **Available on @Android only.** Caches the image that can be later used in ImageView
+   *
+   * @param url The remote location of the image.
+   *
+   * @param callback The function that will be called with the `requestId`. Callback is performed before the prefetch starts.
+   * You can use `abortPrefetch` only after prefetching has started.
+   *
    * @return an empty promise.
    */
-  static async prefetch(url: string): Promise<void> {
-    if (Platform.OS !== 'android') {
+  static async prefetch(url: string, callback?: ImagePrefetchCallback): Promise<void> {
+    if (!ExpoImageModule.abortPrefetch) {
```

Comment:
```suggestion
    if (Platform.OS !== 'android') {
```
Perhaps you accidentally overwritten the correct `if`

Reply:
No, I would like to keep the same convention in both methods. What do you think about it?

Reply:
No problem, but then change `ExpoImageModule.abortPrefetch` to `ExpoImageModule.prefetch` 😉

Reply:
Oh, yes, I missed it.